ai-contribution-policy.md

```diff
@@ -0,0 +1,153 @@
+# KubeVirt AI Contribution Policy
+
+## Overview
+
+This policy establishes guidelines for contributions that involve Artificial
+Intelligence (AI) tools, including but not limited to Large Language Models
+(LLMs), code generation tools, and AI-assisted development environments. This
+is a living document that will evolve as AI technology and legal frameworks
+mature.
+
+## Motivation
+
+AI tools are powerful assistants that can allow developers to become more
+productive when configured and used correctly.
+
+This policy encourages their use within the KubeVirt project to boost both
+productivity and innovation while ensuring transparency. This allows the community
+to learn and refine our policies and practices accordingly in order to maximise
+the value of these tools.
+
+## AI Tool Disclosure Requirements
+
+### Mandatory Disclosure
+
+All contributors **MUST** disclose AI tool use when submitting code,
+documentation, or other content to the KubeVirt project.
+
+Disclosure **MUST** take the form of an `Assisted-by` line within the commit
+attributing the AI tool used, for example:
+
+```text
+Assisted-by: Claude <noreply@anthropic.com>
+```
+
+This will be enforced through the use of the emerging
+[AGENT.md](https://ampcode.com/AGENT.md) standard with symlinks provided to the
```

Comment:
side note: AGENT.md should be part of the .github repository, so it's available on all repositories as default, while it's still overridable with a custom repository configuration.